catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/pom.xml

```diff
@@ -38,24 +38,32 @@
             <artifactId>catalog-core-api-impl</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>javax.ws.rs</groupId>
+            <artifactId>javax.ws.rs-api</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.apache.cxf</groupId>
             <artifactId>cxf-rt-frontend-jaxrs</artifactId>
         </dependency>
         <dependency>
-            <groupId>ddf.jaxb</groupId>
-            <artifactId>spatial-wfs-v1_0_0-schema-bindings</artifactId>
-            <version>${ddf.jaxb.version}</version>
+            <groupId>org.apache.ws.xmlschema</groupId>
+            <artifactId>xmlschema-core</artifactId>
+            <version>2.2.3</version>
```

Comment:
this version could be extracted to the root pom since it is used in more than one place